onboard/src/pages/subsections/backgroundInformation/EmploymentEligbility.jsx

```diff
@@ -1,7 +1,132 @@
 import React from "react";
+import Radio from "@mui/material/Radio";
+import RadioGroup from "@mui/material/RadioGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormControl from "@mui/material/FormControl";
+import FormLabel from "@mui/material/FormLabel";
+import TextField from "@mui/material/TextField";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import Grid from "@mui/material/Grid";
+import Paper from "@mui/material/Paper";
+import Box from "@mui/material/Box";
+
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === "dark" ? "#1A2027" : "#fff",
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  textAlign: "center",
+  color: theme.palette.text.secondary,
+}));
 
 function EmploymentEligbility() {
-  return <></>;
+  return (
+    <>
+      <Box sx={{ width: "100%" }}>
+        <Grid container rowSpacing={1} columnSpacing={{ xs: 1, sm: 2, md: 3 }}>
+          <Grid item xs={6}>
+            <Item>
+              <FormControl>
+                <FormLabel id="18-yrs-or-not">
+                  {" "}
+                  Are you 18 years of age?
+                </FormLabel>
```

Comment:
I like what is being done with all of these fields, however, we need to make controlled inputs so we can use the state for various things. So for instance, to be able to send data to the server & database, or to control whether the user is able to proceed to the next section based on filling out required fields. Look into how to add requirements to fields.